lua/entities/gmod_wire_digitalscreen/init.lua

```diff
@@ -231,9 +231,13 @@ function ENT:WriteCell(Address, value)
 			end
 			self.Memory = mem
 			-- clear pixel data from usermessage queue
-			while #self.ChangedCellRanges > 0 and
-				  self.ChangedCellRanges[1].start + self.ChangedCellRanges[1].length < 1048500 do
```

Comment:
It's worth noting that the previous code had a subtle bug: if you change pixel data then control data then pixel data and then clear screen, then the first pixel data would be cleared out, but the second one wouldn't as it'd stop as soon as it found control data.